### DISCUSSION TITLE
UBL / HS / Templates, compiled firmware questions

### DISCUSSION THREAD

**Discussion by Friday13th87** [Jyers/Marlin#374] [Q&A] [answered]
Hello there.
Is there a reason why there is no config template with UBL+HS+bltouch availible?
And in the firmware release section the smallest UBL mesh is 10x10 - does that mean, does there is just nothing else released or should i use at least a 10x10 mesh for UBL?

What i was wondering is, if it would be better to add a edge frame to the +X side that the bltouch can messure every single point or if it would be better to manually add the missing probe points (on the 10x10 mesh all X9/Y0-9 and on the 15x15 mesh all X14/Y0-14 are missing, as they cant probed automatically)

looking forward for some advice :D

**Comment by Jyers** [comment, marked as answer]
The config templates are fairly general. So setting like high speed probing, which consist of a singular line change, are omitted. Simply enable #define BLTOUCH_HS_MODE in configuration_adv.h for high speed probing.

As far as the mesh question, it is generally best to have the mesh cover the whole bed, and only probe what you can. That way the rest can be inferred and you have a mesh full mesh, rather than one that is cropped to your probing area.
